### ISSUE TITLE
Deprecation warning

### ISSUE THREAD

**Issue by FireFart** [biola/punching_bag#28]
```
DEPRECATION WARNING: Dangerous query method (method whose arguments are used as raw SQL) called with non-attribute argument(s): "SUM(punches.hits) DESC". Non-attribute arguments will be disallowed in Rails 6.0. This method should not be called with user-provided values, such as request parameters or model attributes. Known-safe values can be passed by wrapping them in Arel.sql(). (called from block in _app_views_welcomes_index_html_erb___4028777297065540440_46929003508240 at /app/app/views/welcomes/index.html.erb:43)
```

I think this issue comes from https://github.com/biola/punching_bag/blob/d7f7a2905feedbf0e37140d283a9530f288d63ee/lib/punching_bag/acts_as_punchable.rb#L17

**Comment by adamcrown** [comment]
Thanks @FireFart, and sorry for the long delay in getting to this. I'll see if I can get up a fix.

**Comment by adamcrown** [comment]
I just released version `0.6.1` that should fix this issue. Thanks for reporting it.
